src/style-flight.scss

```
@@ -20,3 +20,8 @@
 slot[name=footer] {
   color: var(--auro-color-text-secondary-on-light);
 }
+
+.header-container {
```

```suggestion
.headerContainer {
```

`.header-container` implies that `container` is a child of `header`. 

https://auro.alaskaair.com/webcorestylesheets/conventions

---

#fixed